openedx/core/lib/block_structure/transformers.py

```diff
@@ -0,0 +1,100 @@
+"""
+Module for a collection of BlockStructureTransformers.
+"""
+from logging import getLogger
+
+from .exceptions import TransformerException
+from .transformer_registry import TransformerRegistry
+
+
+logger = getLogger(__name__)  # pylint: disable=C0103
+
+
+class BlockStructureTransformers(object):
+    """
+    Class for a collection of block structure transformers.
+    """
```

Comment:
Please put in a few notes about how this class should be used. Is it ok to access the `transformers` attribute directly? Mutate it? Can I add to `usage_info`? Is there any point at which initialization is final and further changes will cause issues?